### ISSUE TITLE
【bug】当batch中只有一条数据时，计算RDropLoss会出现nan值

### ISSUE THREAD

**Issue by tianyunzqs** [Tongjilibo/bert4torch#175]
我做的是文本分类任务，构建的基于bert模型的文本分类，用的RDrop loss

```
model = Model(pool_method=pool_method).to(device)
model.compile(
    loss=RdropLoss(),
    optimizer=optim.Adam(model.parameters(), lr=lr),
    metrics=['accuracy']
)
```

我用bert4keras测试了同样的数据划分，好像没有出现这个问题，波神有空可以帮忙看看呢
好像是计算loss的时候，y_pred只有一条数据，就会出现y_pred1为空，导致最终loss为nan
```
y_pred1 = y_pred[1::2]
y_pred2 = y_pred[::2]
```

结果类似这样

```
75/75 [==============================] - 0:07 96ms/step - loss: nan - accuracy: 0.7025    
2025-01-20 10:51:27 - Finish Epoch: 1/10 - loss: nan - accuracy: 0.7025
100%|█████████████████████████████████████████████| 9/9 [00:00<00:00, 40.20it/s]
val_acc: 0.78937, best_val_acc: 0.78937


2025-01-20 10:51:27 - Start  Epoch: 2/10
75/75 [==============================] - 0:06 91ms/step - loss: nan - accuracy: 0.7927 
2025-01-20 10:51:34 - Finish Epoch: 2/10 - loss: nan - accuracy: 0.8023
100%|█████████████████████████████████████████████| 9/9 [00:00<00:00, 40.17it/s]
val_acc: 0.83681, best_val_acc: 0.83681
```
